dev/PagerControl/PagerControl.xaml

```diff
@@ -37,14 +33,20 @@
                                         <Setter Target="FirstPageButton.Visibility" Value="Collapsed"/>
                                     </VisualState.Setters>
                                 </VisualState>
+                                <VisualState x:Name="FirstPageButtonHidden">
```

Comment:
>FirstPageButtonHidden [](start = 53, length = 21)

you probably need a visibility setter here as well

Reply:
The idea is to hide the button, yet let it take up space. When we collapse it, it doesn't take up space. If we only set the opacity to 0, it still gets interactions even though we don't want that. To combat that, we disable the button.